common/common-framework/src/test/java/cn/iocoder/common/framework/util/DateUtilTest.java

```diff
@@ -0,0 +1,56 @@
+package cn.iocoder.common.framework.util;
+
+import org.junit.Test;
+import java.util.Date;
+import java.util.GregorianCalendar;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+public class DateUtilTest {
+
+    @Test
+    public void testAddDate() {
+        assertNull(DateUtil.addDate(0, 0));
+        assertEquals(1_778_410_800_000L, DateUtil.addDate(
+                new Date(1_515_585_600_000L), 2, 100).getTime());
```

Comment:
I think Freddy will say to avoid using current date/time

Reply:
Current date and time isnt beeing used in this section

Reply:
I put this in the wrong place.. supposed to be for the `testIsBetween` -_- my bad